tdm-be/src/templates/processing-error.njk

```diff
@@ -4,6 +4,10 @@ Vous trouverez dans ce mail un rapport d'erreur du traitement {{ processingId }}
 
 {{ errorMessage }}
 
+Vous pouvais retrouvé une description de la cause possible de l'erreur sur le lien ci-dessous :
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
Vous pouvez retrouver une description de la cause possible de l'erreur sur le lien ci-dessous :
```